config/fonts/Metropolis/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2011 The Metrophobic Project Authors (https://github.com/googlefonts/MetrophobicFont), with Reserved Font Name ‘Metrophobic’.

```

Comment:
This license text is from the wrong font, copy from here: https://raw.githubusercontent.com/openmaptiles/fonts/master/metropolis/SIL%20Open%20Font%20License.txt